app/views/comments/_edit.html.erb

```diff
@@ -1,146 +1,152 @@
+          else $('#node_images').val(data.result.id)
+        },
 
-          // fileuploadfail: function(e,data) {},
-          progressall: function (e, data) {
-            var progress = parseInt(data.loaded / data.total * 100, 10);
-            $('#c<%= comment.id%>progress #c<%= comment.id%>progress-bar').css(
-                'width',
-                progress + '%'
-            );
-          }
-        });
-      </script>
+        // fileuploadfail: function(e,data) {},
+        progressall: function (e, data) {
+          var progress = parseInt(data.loaded / data.total * 100, 10);
+          $('#c<%= comment.id%>progress #c<%= comment.id%>progress-bar').css(
+              'width',
+              progress + '%'
+          );
+        }
+      });
+    </script>
 
-      <div class="comment-preview well col-md-11" id="c<%= comment.id %>preview" style="background:white;display: none">
-      </div>
+    <div class="comment-preview well col-md-11" id="c<%= comment.id %>preview" style="background:white;display: none">
+    </div>
 
-      <div class="control-group">
-        <button type="submit" class="btn btn-primary"><%= translation('comments._edit.publish') %></button>
-          <a class="btn btn-default preview-btn"  data-previewing-text="Hide Preview"
-              onClick="$('#c<%= comment.id %>preview').toggle();
-              $('#c<%= comment.id %>text').toggle();
-              $('#c<%= comment.id %>text').next('#imagebar').toggle();
-              this.previewing = !this.previewing;
-              $('#c<%= comment.id %>edit .preview-btn').button(this.previewing ? 'previewing' : 'reset');
-              $E.generate_preview('c<%= comment.id %>preview',$('#c<%= comment.id %>text').val())">
-            Preview
-          </a>
-          <a class="btn btn-default" onClick="$('#c<%= comment.id %>show').toggle();$('#c<%= comment.id %>edit').toggle()">
-            Cancel
-          </a>
+    <div class="control-group">
+      <button type="submit" class="btn btn-primary"><%= translation('comments._edit.publish') %></button>
+        <a class="btn btn-default preview-btn"  data-previewing-text="Hide Preview"
+            onClick="$('#c<%= comment.id %>preview').toggle();
+            $('#c<%= comment.id %>text').toggle();
+            $('#c<%= comment.id %>text').next('#imagebar').toggle();
+            this.previewing = !this.previewing;
+            $('#c<%= comment.id %>edit .preview-btn').button(this.previewing ? 'previewing' : 'reset');
+            $E.generate_preview('c<%= comment.id %>preview',$('#c<%= comment.id %>text').val())">
+          Preview
+        </a>
+        <a 
+          class="btn btn-default" 
+          onClick="
+            $('#edit-comment-form-wrapper-<%= comment.id %>').toggle();
+            $('#<%= comment.id %>-like-emojis').toggle();
+            $('#c<%= comment.id %>edit').toggle();
```

Comment:
I also made a change here to toggle the comment's emoji reactions when opening an Edit Comment form. I think it's a lot less confusing.

**Before**: (emoji heart reaction is still present):
https://user-images.githubusercontent.com/4361605/105804161-f1e25b80-5f53-11eb-97c7-cccbba252114.mov

**After**:
https://user-images.githubusercontent.com/4361605/105804185-00307780-5f54-11eb-98f7-db9e6540767e.mov